actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -89,10 +107,56 @@ pub fn codecopy(state: &mut ExecutionState, code: &[u8]) -> Result<(), StatusCod
 
 #[inline]
 pub fn call<'r, BS: Blockstore, RT: Runtime<BS>>(
-    _state: &mut ExecutionState,
+    state: &mut ExecutionState,
     _platform: &'r System<'r, BS, RT>,
-    _kind: CallKind,
-    _is_static: bool,
+    kind: CallKind,
+    is_static: bool,
 ) -> Result<(), StatusCode> {
+    let ExecutionState { stack, memory, .. } = state;
+
+    let gas = stack.pop();
+    let dst: H160 = crate::interpreter::uints::_u256_to_address(stack.pop());
+    let value = if is_static || matches!(kind, CallKind::DelegateCall) {
+        U256::zero()
+    } else {
+        stack.pop()
+    };
+    let has_value = !value.is_zero();
+    let input_offset = stack.pop();
+    let input_size = stack.pop();
+    let output_offset = stack.pop();
+    let output_size = stack.pop();
+
+    stack.push(U256::zero()); // Assume failure. TODO wha
```

Comment:
TODO was me being confused why this was a thing, I was expecting it was something weird with EVM @vyzo u know of any reason that this would be needed?